code/game/gamemodes/clock_cult/clock_helpers/resource_helpers.dm

```diff
@@ -0,0 +1,19 @@
+//Helpers for potential and wisdom.
+
+//Returns true if the potential is greater than or equal to a set amount, and false otherwise.
+/proc/has_clockwork_potential(amt)
```

Comment:
I must've missed the memo, then. What should they be? Should I just refer to the var directly?